src/components/Alteration/Articles/CompanyProvisions.vue

```diff
@@ -4,54 +4,86 @@
       <v-row no-gutters>
         <v-col cols="3">
           <label class="define-company-provisions-title">Pre-existing<br>Company Provisions</label>
-          <v-chip v-if="hasProvisionsRemovedPropsChanged" x-small label color="primary" text-color="white">
+          <v-chip
+            v-if="hasProvisionsRemovedPropsChanged"
+            x-small
+            label
+            color="primary"
+            text-color="white"
+          >
             {{ getEditedLabel }}
           </v-chip>
         </v-col>
 
-        <v-col v-if="provisionsRemoved" cols="7" id="none-of-provisions-apply-text" class="info-text">
+        <v-col
+          v-if="provisionsRemoved"
+          id="none-of-provisions-apply-text"
+          cols="7"
+          class="info-text"
+        >
           {{ companyResolvedText }}
         </v-col>
-        <v-col v-else cols="7" id="has-pre-existing-provisions-text" class="info-text">
+        <v-col
+          v-else
+          id="has-pre-existing-provisions-text"
+          cols="7"
+          class="info-text"
+        >
           This company has Pre-existing Company Provisions.
         </v-col>
 
-        <v-col v-if="!hasProvisionsRemovedPropsChanged" cols="2" class="mt-n2 align-right">
+        <v-col
+          v-if="!hasProvisionsRemovedPropsChanged"
+          cols="2"
+          class="mt-n2 align-right"
+        >
           <v-btn
             id="change-company-provisions"
             text
             color="primary"
             @click="isEditing = true"
           >
-            <v-icon small>mdi-pencil</v-icon>
+            <v-icon small>
+              mdi-pencil
+            </v-icon>
             <span>{{ getEditLabel }}</span>
           </v-btn>
         </v-col>
-        <v-col v-else cols="2" class="pt-0 mt-n2 align-right">
+        <v-col
+          v-else
+          cols="2"
+          class="pt-0 mt-n2 align-right"
+        >
           <v-btn
             id="undo-company-provisions"
             text
             color="primary"
             class="undo-company-provisions"
             @click="resetCompanyProvisions"
           >
-            <v-icon small>mdi-undo</v-icon>
+            <v-icon small>
+              mdi-undo
+            </v-icon>
             <span>Undo</span>
           </v-btn>
 
           <!-- More Actions Menu -->
           <span class="more-actions">
             <v-menu
-              offset-y left nudge-bottom="4"
               v-model="dropdown"
+              offset-y
+              left
+              nudge-bottom="4"
             >
-              <template v-slot:activator="{ on }">
+              <template #activator="{ on }">
```

Comment:
I need to test this.

Reply:
Works.

![image](https://user-images.githubusercontent.com/10002889/234069450-b737e77c-3fdd-4fbe-9bb1-70bd685b8ae8.png)